### DISCUSSION TITLE
Mac 8.5.9 - Massive memory consumption during Google Drive transfer

### DISCUSSION THREAD

**Discussion by tcj** [iterate-ch/cyberduck#14540] [Q&A]
Hi — haven't used Cyberduck in a few years.  Tonight I'm helping a friend mirror a Google Drive archive, and remembered Cyberduck's support for cloud storage.  Using an 8 GB iMac on Monterey 12.6.3.

Cyberduck seems to consume about 1 GB of RAM every 2-3 minutes.  It's at 37.90 GB now.  When I first noticed the memory consumption on this transfer, it was at closer to 4 GB.

This transfer has currently downloaded 94.8 GB of 129.1 GB, or 73%, at 15.1 MB /sec, and says 37 minutes remaining.

To me this seems like a memory leak or something, but I recognize these issues can be more complex.  It does not matter which windows I have open, the memory use keeps climbing.

In the time it took to write this, memory consumption is up to 39.14 GB.

Any chance an earlier version or macOS downgrade would be helpful to keep memory use down?

**Comment by tatze96** [comment]
Do you still see the described behavior within the current release version (8.6.0)?
